new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/InstallWITCommon.java

```diff
@@ -42,17 +36,17 @@ protected void checkDirectory(String dir) {
   }
 
   /**
-   * Check if the required directoies exist.
-   * Currently the directies will be created if missing. We may remove this function
-   * once we have all required working directies pre-created.
+   * Check if the required directories exist.
+   * Currently the directories will be created if missing. We may remove this function
+   * once we have all required working directories pre-created.
    *
    * @param dir the directory that needs to be checked
    */
   protected void checkFile(String fileName) throws FileNotFoundException {
     File file = new File(fileName);
     if (!file.exists()) {
```

Comment:
do we need to check that it is a file, not a directory or something else?

Reply:
good point. I need to check if it is indeed a file.